### ISSUE TITLE
Ask for user and password

### ISSUE THREAD

**Issue by jaskolowski** [Athlon1600/php-proxy-app#152]
Hello,
Iḿ trying to configure a private proxy to allow teachers navigate the OPAC from out side of work.
Is there a whay to set up a login where the user have to put a password?
thanks

**Comment by Benji-Collins** [comment]
There are a number of ways this could be done, but the easiest is probably to set up a login with Apache. Obviously these instruction will be different if you use a different version or web server but it should get you on the right track.

https://www.digitalocean.com/community/tutorials/how-to-set-up-password-authentication-with-apache-on-ubuntu-14-04


________________________________

**Comment by jaskolowski** [comment]
Well,
 I have tried that. and at the fail I think to ask here.
This is mi enviroment.... 
the main server is ubuntu 16.04 and the proxy is in a container lxd based in a daily image.
I did the installation of php-proxy with composer in a clean apache2 and later I tried what that ocean tutorial describes with a total failure. I checked the logs but there are no signs of error.

So, I think that, there may be a simple way to set up index.php to anticipate login to the proxy. or a plugin
thanks for your answer
